Cargo.toml

```diff
@@ -47,7 +47,7 @@ tokio = { version = "1.25.0", features = ["full"] }
 pretty_assertions = "1.0.0"
 
 [features]
-default = ["rust"]
+default = ["html", "rust"]
```

Comment:
Otherwise you have to remember to include it during development. We might as well just include it all the time.